atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownDeleteTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.cassandra.multinode;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import com.google.common.collect.ImmutableMultimap;
+import com.palantir.common.exception.PalantirRuntimeException;
+
+public class OneNodeDownDeleteTest {
+
+    @Rule
+    public ExpectedException expectException = ExpectedException.none();
+
+    @Test
+    public void deletingThrows() {
+        expectException.expect(PalantirRuntimeException.class);
+        OneNodeDownTestSuite.db.delete(OneNodeDownTestSuite.TEST_TABLE,
```

Comment:
Elsewhere we do this with 

```java
import static org.assertj.core.api.Assertions.assertThatThrownBy;

...

        assertThatThrownBy(() -> myFunction(myArgument)))
                .isInstanceOf(MyException.class);
```

Apparently the output is a bit nicer from this.